packages/standard-minifiers-css/plugin/minify-css.js

```diff
@@ -14,17 +14,19 @@ CssToolsMinifier.prototype.processFilesForBundle = function (files, options) {
 
   if (! files.length) return;
 
-  var merged = mergeCss(files);
-
+  // Don't merge or minify anything for development
   if (mode === 'development') {
-    files[0].addStylesheet({
-      data: merged.code,
-      sourceMap: merged.sourceMap,
-      path: 'merged-stylesheets.css'
+    files.forEach(function (file) {
+      file.addStylesheet({
+        data: file.getContentsAsBuffer(),
+        sourceMap: file.getSourceMap(),
+        path: file.getPathInBundle()
```

Comment:
Yep we use Semantic UI with custom theming, so we have the raw LESS files in our source tree. This is the default behavior for the semantic:ui package so it's pretty common.